src/main/anovos/feature_recommender/featrec_init.py

```diff
@@ -1,6 +1,70 @@
-from anovos.feature_recommender.feature_exploration import *
+from sentence_transformers import SentenceTransformer
 from re import finditer
+import pandas as pd
+import numpy as np
 import copy
+import warnings
+
+
+warnings.filterwarnings("ignore")
+
+
+def init_model():
+    """
+
+    Returns
+    -------
+    Semantic model used in Feature Explorer and Recommender
+    """
+    model = SentenceTransformer("all-mpnet-base-v2")
+    return model
```

Comment:
```suggestion
class _TransformerModel:
    def __init__(self):
        self._model = None

    @property
    def model(self) -> SentenceTransformer:
        if self._model is None:
            self._model = SentenceTransformer("all-mpnet-base-v2")
        return self._model

model = _TransformerModel()
```

I wouldn't load the model every time, but exactly once. Loading such a huge model even from disk can cause significant delay.